examples/webhook-signing/node-koa/koa.js

```diff
@@ -0,0 +1,65 @@
+const Stripe = require('stripe');
```

Comment:
this is a js file but you have a tsconfig is this intended?

Reply:
no